.github/workflows/releaser.yaml

```diff
@@ -0,0 +1,52 @@
+name: release
+
+on:
+  push:
+    branches:
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        go-version: [1.20.x]
+        os: [macos-latest, ubuntu-latest]
```

Comment:
Can we add Windows here?

Also since mac doesn't have TPM, the usage will be limited.

Reply:
Given the ci.yaml has "todo: get this working on windows-latest", I'll leave that for a future PR from Alex.

Reply:
sgtm